packages/client/src/StreamrClientError.ts

```diff
@@ -0,0 +1,7 @@
+export class StreamrClientError extends Error {
+    constructor(message: string, readonly errorCode?: 'NO_STORAGE_NODES') {
+        super(message)
+        Error.captureStackTrace(this, StreamrClientError)
+        this.name = this.constructor.name
```

Comment:
Is `this.name` a convention of needed for some specific purpose?

Reply:
It affects the error message that gets printed. If you don't set the name it will be `Error: something went wrong` vs. `StreamrClientError: something went wrong`.